modules/tinymce/src/core/main/ts/content/InsertContentImpl.ts

```diff
@@ -181,16 +181,19 @@ const moveSelectionToMarker = (editor: Editor, marker: HTMLElement | null): void
   dom.remove(marker);
 
   if (parentBlock && dom.isEmpty(parentBlock)) {
+    const isCell = isTableCell(parentBlock);
+
     Remove.empty(SugarElement.fromDom(parentBlock));
 
     rng.setStart(parentBlock, 0);
     rng.setEnd(parentBlock, 0);
 
-    if (!isTableCell(parentBlock) && !isPartOfFragment(parentBlock) && (nextRng = findNextCaretRng(rng))) {
+    if (!isCell && !isPartOfFragment(parentBlock) && (nextRng = findNextCaretRng(rng))) {
       rng = nextRng;
       dom.remove(parentBlock);
     } else {
-      dom.add(parentBlock, dom.create('br', { 'data-mce-bogus': '1' }));
+      // If parentBlock is a table cell, add a br without 'data-mce-bogus' attribute.
```

Comment:
Add the Jira ticket number to this comment. Instead of stating the obvious, explain that it is to keep it consistent with the <br> in other empty table cells and for `remove_trailing_brs` to function correctly.

Reply:
```suggestion
      // TINY-9860: If parentBlock is a table cell, add a br without 'data-mce-bogus' attribute.
```